app/models/faq_item.rb

```diff
@@ -0,0 +1,29 @@
+class FaqItem < FrozenRecord::Base
+  include ActionView::Helpers::UrlHelper
+  include Rails.application.routes.url_helpers
+
+  add_index :title, unique: true
+
+  def id
+    @id ||= "collapse-#{title.parameterize}"
+  end
+
+  def body
+    @body ||= begin
+      body_md = ERB.new(body_md_erb).result(binding)
+      Kramdown::Document.new(body_md).to_html.html_safe
+    end
+  end
+
+  private
+
+  def controller
+    @controller ||= begin
+      controller = ApplicationController.new
+      controller.define_singleton_method(:url_options) do
+        Rails.application.config.action_mailer.default_url_options
```

Comment:
Out of curiosity, what is that for ?

Reply:
The `link_to` helper needs to have access to host, port, etc. to build links.